.cicd/helpers/execute_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+if [[ "$BUILDKITE" == 'true' ]]; then
+  apt-get update && apt-get -y install wget
```

Comment:
I understand that it is running only on Ubuntu currently but for future, I suggest to do platform assert here.
like 
`if [[ arch == "Ubuntu"]]; then`
`#do magic`
`else `
`echo "unsupported architecture"`
`exit 1`
`fi`
something similar to:
https://github.com/EOSIO/eos/blob/develop/scripts/helpers/eosio.sh

Reply:
I tried to do this, unfortunately it looks like the environment variables to determine the OS are not set the same way as EOSIO (i got an error), so we may require additional effort to make this work, but i will keep in mind to add this in the future